lib/db_pool_web/controllers/api/database_controller.ex

```diff
@@ -18,6 +18,13 @@ defmodule DbPoolWeb.API.DatabaseController do
     json(conn, :ok)
   end
 
+  def reimport(conn, %{"id" => id}) do
+    database = Core.get_database!(id)
+    { :ok, _database } = Core.import_dump_to_database(database)
```

Comment:
will it conflict with the existing database? Should we clean previous data first ? I believe you'd have tested this already.

Reply:
I had the same doubt. @iffyuva said this should be fine. He said, this is what he is doing for imports manually via console 🤔

Reply:
https://raw.githubusercontent.com/pthom/northwind_psql/master/northwind.sql

So if we take a look at this psql dump, we can actually see that there are drops before create. 🙂